spec/models/hathi_trust_links_spec.rb

```diff
@@ -96,21 +97,23 @@
       end
 
       it 'is the url to the item' do
-        expect(ht_links.url).to include('hathitrust.org/cgi/pt?id=1234567&view=1up')
+        expect(ht_links.url).to include('hathitrust.org/cgi/pt?id=1234567')
       end
 
       context 'when publicly_available' do
         it 'passes the item URL through the target param' do
-          expect(ht_links.url).to include('target=https://babel.hathitrust.org/cgi/pt?id=1234567&view=1up')
+          expect(ht_links.url).to include('target=https://babel.hathitrust.org/cgi/pt?id=1234567')
         end
       end
 
       context 'when not publicly_available' do
-        before { document_data[:ht_access_sim] = ['access'] }
+        before do
+          document_data[:ht_access_sim] = ['deny']
+          document_data[:ht_htid_ssim] = ['loc.ark:/13960/t17m0qv0b']
```

Comment:
Test URLs with special characters and set it up properly for testing not public URLs (was not done correctly before).